task0/LinearRegression_beyzaNur.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>You may print results in a more human readable format:</p><pre class="ql-syntax" spellcheck="false">score = model.score(ageTransformed,Price.reshape(-1,1))
print(f"The regression score is ~ {score:.4f}")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>